### ISSUE TITLE
Repeated measure ANOVA

### ISSUE THREAD

**Issue by eeguser** [Mensen/ept_TFCE-matlab#14]
Hi

I'm first conducting a repeated measure ANOVA 1 x 5 with data structured in cell array as TFCE function use: 
5 x 1 cell array 
The cell array comprise: 
17 x 64 x 358
17 x 64 x 358
17 x 64 x 358
17 x 64 x 358
17 x 64 x 358

Each cell is a condition. 

How do I implement that in your toolbox? 
The ept_TFCE_ANOVA takes in two data sets - that would be for my next analysis with two groups (same paradigm). 

The next analysis is to do a 2 x 5 mixed ANOVA. 

Any help is much appreciated.

**Comment by Mensen** [comment]
Indeed... the for the ANOVA it always expects to have 2 factors... so if you want to run a one-way repeated measures ANOVA... the "trick" is to just repmat the Data and pretend you have two factors but the second one is just the same data...

This can be accomplished using the following quick bit of code:
 `Data = repmat(Data, 1, 2);` 
You may have to switch the '2' and the '1' depending on which way the Data is organised...
and then just running the 2x5 ANOVA:
`ept_TFCE_ANOVA(Data, e_loc);`

Then in the results you'll see that one factor basically has a p-value of 1 across the board... but the other should have some mixed results which actually show the ANOVA results between the 5 conditions.

Take care and good luck with your analyses!
Armand
